### ISSUE TITLE
User group is unable to add new fields to form

### ISSUE THREAD

**Issue by jrrdnx** [barrelstrength/craft-sprout-forms#468]
### Description

We have a Client Administrator group with full access both to Sprout Forms as well as access to all individual Sprout Forms permissions (the only exception is Edit Form Entries).

When adding a new field to any form, they receive a "A server error occurred" popup at the top of the screen and are unable to edit the field's detail or save the form.

<img width="1122" alt="Screen Shot 2020-04-17 at 4 59 41 PM" src="https://user-images.githubusercontent.com/4584671/80959288-7f818800-8dd4-11ea-8266-646baad13a11.png">

I have actually seen this error message before when logged in as an Admin, but am unable to reproduce the issue consistently.

Combing the logs, I'm seeing the following error around that timestamp:

```
2020-04-17 16:59:12 [-][28501][-][error][yii\web\HttpException:403] yii\web\ForbiddenHttpException: User is not permitted to perform this action in /home/domaincom/domain.com/vendor/craftcms/cms/src/web/Controller.php:303
Stack trace:
#0 /home/domaincom/domain.com/vendor/barrelstrength/sprout-forms/src/controllers/FieldsController.php(70): craft\web\Controller->requirePermission()
#1 [internal function]: barrelstrength\sproutforms\controllers\FieldsController->actionCreateField()
#2 /home/domaincom/domain.com/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#3 /home/domaincom/domain.com/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams()
#4 /home/domaincom/domain.com/vendor/craftcms/cms/src/web/Controller.php(178): yii\base\Controller->runAction()
#5 /home/domaincom/domain.com/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction()
#6 /home/domaincom/domain.com/vendor/craftcms/cms/src/web/Application.php(291): yii\base\Module->runAction()
#7 /home/domaincom/domain.com/vendor/craftcms/cms/src/web/Application.php(559): craft\web\Application->runAction()
#8 /home/domaincom/domain.com/vendor/craftcms/cms/src/web/Application.php(270): craft\web\Application->_processActionRequest()
#9 /home/domaincom/domain.com/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest()
#10 /home/domaincom/domain.com/web/index.php(24): yii\base\Application->run()
#11 {main}
2020-04-17 16:59:12 [-][28501][-][info][application] $_GET = [
    'p' => 'admin/actions/sprout-forms/fields/create-field'
]

$_POST = [
    'type' => 'barrelstrength\\sproutforms\\fields\\formfields\\Name'
    'formId' => '39883'
    'tabId' => '1207'
]
```

### Steps to reproduce

1. Create a new form / edit an existing form
2. Add a new field

### Additional info

- Craft version: Craft Pro 3.4.17.1
- PHP version: 7.4.2
- Database driver & version: MySQL 5.7.28
- Plugins & versions: 3.11.1

**Comment by BenParizek** [comment]
Thanks @jrrdnx.

I'm able to recreate. Will let you know when we have a fix out.

**Comment by BenParizek** [comment]
@jrrdnx We just released v3.11.3 which should address this issue. Thanks for the detailed report.
